Automattic/jetpack
# Add Woo Express, Business Trial, and 100-Year plan constants

Adds constants and WPCOM plan slug definitions for recently released WPCOM plans: Woo Express Trial, Woo Express Essentials, Woo Express Essentials: Monthly, Woo Express Performance, Woo Express Performance: Monthly, WordPress.com Migration Trial, WordPress.com Business Trial, WordPress.com 100-Year Plan





Fixes #33734

## Proposed changes:

* Adds WoA plan slugs for recently released WPCOM plans (see above), and defines correct plan icons
Currently if a recently released plan is active, the plan icon is not defined, and results in infinite loading of the Jetpack > Plans page in WP-Admin on WoA site.

### Other information:


## Jetpack product discussion


#33734

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Install the Jetpack Beta Tester plugin
* Use Jetpack Beta Tester plugin to find and activate this branch add/woo-express-wpcom-trial-100-year-plan-icons
* Add an affected plan:
```
- Woo Express Trial
- Woo Express Essentials
- Woo Express Essentials: Monthly
- Woo Express Performance
- Woo Express Performance: Monthly
- WordPress.com Migration Trial
- WordPress.com Business Trial
- WordPress.com 100-Year Plan
```
*  Deactivate Jetpack, reactivate Jetpack (reconnect if necessary), and ensure the correct plan slug is showing in WP-CLI:
```
- 'ecommerce-trial-bundle-monthly'
- 'wooexpress-small-bundle-yearly'
- 'wooexpress-small-bundle-monthly'
- 'wooexpress-medium-bundle-yearly'
- 'wooexpress-medium-bundle-monthly'
- 'wp_bundle_migration_trial_monthly'
- 'wp_bundle_hosting_trial_monthly'
- 'wp_com_hundred_year_bundle_centennially'
```
* Navigate to Jetpack > Plans in WP-Admin, and ensure that the plans page loads
